LG-2727: Add "forget all devices" functionality

zachmargolis opened the pull request:
**Why**: So that users can forget devices they may
have remembered accidentally

(This is still waiting on final copy)

**Account page:** New "Forget all devices" link at the bottom of the list of devices

<img width="818" alt="Screen Shot 2020-03-11 at 5 37 38 PM" src="https://user-images.githubusercontent.com/458784/76476393-1fb3c580-63bf-11ea-9ed6-4690fdaaa417.png">
<img width="379" alt="Screen Shot 2020-03-11 at 5 37 44 PM" src="https://user-images.githubusercontent.com/458784/76476400-22aeb600-63bf-11ea-8509-bac3a39e0457.png">

**New screen**: confirmation of forgetting all devices

<img width="733" alt="Screen Shot 2020-03-11 at 5 37 55 PM" src="https://user-images.githubusercontent.com/458784/76476403-26423d00-63bf-11ea-8d9f-7a5666b30e12.png">
<img width="352" alt="Screen Shot 2020-03-11 at 5 37 50 PM" src="https://user-images.githubusercontent.com/458784/76476414-2b9f8780-63bf-11ea-98b7-ec2ef98b1c75.png">

jmhooper commented on app/controllers/users/forget_all_devices_controller.rb:
```diff
@@ -0,0 +1,18 @@
+module Users
+  class ForgetAllDevicesController < ApplicationController
+    before_action :authenticate_user!
```
Doesn't `confirm_two_factor_authenticated` call this?

zachmargolis replied:
Honestly I didn't check, it was just some copy-pasta from another controller. I will dig in and confirm

jmhooper replied:
Looked it up. Here's where it is called in `confirm_two_factor_authenticated`: https://github.com/18F/identity-idp/blob/master/app/controllers/application_controller.rb#L216

zachmargolis replied:
🙈 thanks! fixing

zachmargolis commented:
Some design updates:

- Moved link to its own section on the account page
- Renamed to "Forget all _browsers_ ~devices~" to mirror the "Remember this browser" copy

<img width="842" alt="Screen Shot 2020-03-13 at 11 13 10 AM" src="https://user-images.githubusercontent.com/458784/76649539-2016a280-651e-11ea-86d0-4c6b27bc5820.png">

<img width="654" alt="Screen Shot 2020-03-13 at 11 12 59 AM" src="https://user-images.githubusercontent.com/458784/76649542-21e06600-651e-11ea-8f64-8476d4ac88de.png">
